scripts/tools/generate-integrations-json.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import json
+import os
+import re
+from glob import glob
+
+import yaml
+
+import airflow
+
+if __name__ != "__main__":
+    raise Exception(
+        "This file is intended to be executed as an executable program. You cannot use it as a module."
+        "To run this script, run the './generate-integrations-json.py >integrations.json' command"
+    )
+
+AIRFLOW_ROOT = os.path.abspath(os.path.join(os.path.dirname(airflow.__file__), os.pardir))
+
+result_integrations = []
+for provider_file in glob(f"{AIRFLOW_ROOT}/airflow/providers/**/provider.yaml", recursive=True):
+    with open(provider_file) as f:
+        provider_info = yaml.load(f, Loader=yaml.CLoader)
+        for integration in provider_info.get('integrations', []):
+            doc_url = integration.get("how-to-guide")
+            if doc_url:
+                doc_url = doc_url[0].strip()
+                doc_url = re.sub(f'/{provider_info["package-name"]}/', r"\g<0>stable/", doc_url)
+                doc_url = re.sub(r'\.rst', '.html', doc_url)
+            else:
+                doc_url = f"/docs/{provider_info['package-name'].lower()}/stable/index.html"
+            logo = integration.get("logo")
+
+            result = {
+                'name': integration['integration-name'],
+                'url': doc_url,
+            }
+            if logo:
+                result['logo'] = logo
+            result_integrations.append(result)
+
+print(
```

Comment:
Ideally, this script should update the necessary files in our target repository - `aiirflow-site`. In the next step, we can create a PR that will contain all the necessary changes, or add it to our CI pepline.
See example: 
https://github.com/apache/airflow/blob/master/docs/publish_docs.py
Doc: https://github.com/apache/airflow/blob/master/dev/README_RELEASE_PROVIDER_PACKAGES.md#publish-documentation
https://github.com/apache/airflow/blob/master/dev/README_RELEASE_AIRFLOW.md#publish-documentation